src/pages/Liked/Liked.jsx

```diff
@@ -1,5 +1,36 @@
 import React from "react";
+import { Navbar, Sidebar, VideoCard } from "components";
+import styles from "./liked.module.css";
+import { useVideosOperations } from "context";
 
 export function Liked() {
-  return <div>Liked</div>;
+  const {
+    videosOperations: { likedVideos },
+  } = useVideosOperations();
+
+  return (
+    <div className={styles.container}>
+      <section className={styles.navbar}>
+        <Navbar />
+      </section>
+      <section className={styles.sidebar}>
+        <Sidebar />
+      </section>
+
+      <main className={styles.main}>
+        <div className={styles.topBar}>
+          {likedVideos.length > 0 ? (
```

Comment:
Suggestion - You can make this more simplied , instead of likedVideos.length > 0 ? ( ) : ( ) , you can try likedVideos.length ? ( ) : ( )

Reply:
sure will make the changes. Thanks for pointing out.